src/Movie.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+
+function MovieDetails(props) {
```

Comment:
Destructuring here will save you having to use props in component and will make it easier to see what props are in use